openwisp_utils/admin_theme/templates/admin/base_site.html

```diff
@@ -14,6 +14,16 @@
     <link rel="stylesheet" type="text/css" href="{% static "admin/css/openwisp.css" %}" />
     <link rel="icon" type="image/x-icon" href="{% static "ui/openwisp/images/favicon.png" %}" />
 {% endif %}
+{% if OPENWISP_TEST_MODE %}
+    <link rel="stylesheet" type="text/css" href="{% static "admin/css/openwisp.css" %}" />
+    <link rel="icon" type="image/x-icon" href="{% static "ui/openwisp/images/favicon.png" %}" />
```

Comment:
I've not tested locally, but **openwisp.css** and **favicon.png** are duplicated while testing?